src/cli/commands/gsn-registry.ts

```diff
@@ -54,8 +55,9 @@ function formatVersion (id: string, versionInfo: VersionInfo, showDate = false):
 (async () => {
   const nodeURL = getNetworkUrl(commander.network)
 
+  const logger = createLogger('debug')
```

Comment:
commands logger might get commander, so we can define cmdline option '--debug' to apply to all.

Reply:
Fixed